pages/index.tsx

```diff
@@ -1,16 +1,38 @@
 import Head from 'next/head';
+import Link from 'next/link';
+import { useUser } from '@auth0/nextjs-auth0/client';
+import { useState } from 'react';
+
+export default function Home(this: any) {
+  const { user, error, isLoading } = useUser();
```

Comment:
Since `error` and `isLoading` are currently unused, they can be removed.
```suggestion
  const { user } = useUser();
```